metafacture-runner/src/main/dist/examples/transform/pica-to-marcxml/pica-to-marcxml_fix.flux

```diff
@@ -0,0 +1,13 @@
+// opens file 'fileName', interprets the content as non normalized serialized
+// pica+, encode as marc, morphs to marcxml and writes to standard out
+
+default fileName = FLUX_DIR + "nonNormalized.pica";
+
+fileName
+| open-file
+| as-lines
+| lines-to-records
+| decode-pica(normalizedSerialization="false", ignoreMissingIdn="true")
+| fix(FLUX_DIR + "morph-pica-to-marcxml.fix")
+| encode-marcxml(ensurecorrectmarc21xml="true")
```

Comment:
The Morph example uses `encode-marc21` + `decode-marc21` + `encode-marcxml` w/o `ensurecorrectmarc21xml`.

Reply:
Related to: https://github.com/metafacture/metafacture-core/pull/662#discussion_r1941528758

Reply:
Fixed by: https://github.com/metafacture/metafacture-core/pull/663/commits/77e15f31b047b2a133e11718d88888052620f5b2